libcontainer/configs/validate/validator_test.go

```diff
@@ -504,103 +504,108 @@ func TestValidateIDMapMounts(t *testing.T) {
 		config *configs.Config
 	}{
 		{
-			name:  "idmap mount without bind opt specified",
```

Comment:
Can we retain existing tests?

Reply:
I haven't removed any tests. The diff makes it look weird, but this just changes the name of the test, and makes it actually test a realistic configuration (it has a valid `Device` value).